pages/dao/[symbol]/proposal/components/instructions/ProposalForm.tsx

```diff
@@ -0,0 +1,203 @@
+            index={index}
+            governedAccount={governedAccount}
+          />
+        )
+      case Instructions.SolendRefreshObligation:
+        return (
+          <SolendRefreshObligation
+            index={index}
+            governedAccount={governedAccount}
+          />
+        )
+      case Instructions.SolendRefreshReserve:
+        return (
+          <SolendRefreshReserve
+            index={index}
+            governedAccount={governedAccount}
+          />
+        )
+      case Instructions.SolendWithdrawObligationCollateralAndRedeemReserveLiquidity:
+        return (
+          <SolendWithdrawObligationCollateralAndRedeemReserveLiquidity
+            index={index}
+            governedAccount={governedAccount}
+          />
+        )
+      case Instructions.RaydiumAddLiquidity:
+        return (
+          <RaydiumAddLiquidityToPool
+            index={index}
+            governedAccount={governedAccount}
+          />
+        )
+      case Instructions.RaydiumRemoveLiquidity:
+        return (
+          <RaydiumRemoveLiquidityFromPool
+            index={index}
+            governedAccount={governedAccount}
+          />
+        )
+      case Instructions.UXDInitializeController:
+        return <UXDInitializeController index={index} governance={governance} />
+      case Instructions.UXDSetRedeemableGlobalSupplyCap:
+        return (
+          <UXDSetRedeemableGlobalSupplyCap
+            index={index}
+            governance={governance}
+          />
+        )
+      case Instructions.UXDSetMangoDepositoriesRedeemableSoftCap:
+        return (
+          <UXDSetMangoDepositoriesRedeemableSoftCap
+            index={index}
+            governance={governance}
+          />
+        )
+      case Instructions.UXDRegisterMangoDepository:
+        return (
+          <UXDRegisterMangoDeposiory index={index} governance={governance} />
+        )
+      case Instructions.UXDDepositInsuranceToMangoDepository:
+        return (
+          <UXDDepositInsuranceToMangoDepository
+            index={index}
+            governance={governance}
+          />
+        )
+      case Instructions.UXDWithdrawInsuranceFromMangoDepository:
+        return (
+          <UXDWithdrawInsuranceFromMangoDepository
+            index={index}
+            governance={governance}
+          />
+        )
+      case Instructions.Mint:
+        return <Mint index={index} governance={governance} />
+      case Instructions.Base64:
+        return <CustomBase64 index={index} governance={governance} />
+      case Instructions.None:
+        return <Empty index={index} governance={governance} />
+      case Instructions.Grant:
+        return <Grant index={index} governance={governance} />
+      case Instructions.Clawback:
+        return <Clawback index={index} governance={governance} />
+      default:
+        null
+    }
+  }
+
+  return (
+    <>
+      {![
+        Instructions.Transfer,
```

Comment:
move this list to its own variable in another file ?

Reply:
I would prefer to keep that here.
The objective being to make it disappear over time, I find it a nice incentive to have that ugly list screaming at your face everytime you open that file